base/compiler/ssair/inlining2.jl

```diff
@@ -577,15 +577,17 @@ function analyze_method!(idx, f, ft, metharg, methsp, method, stmt, atypes, sv,
     end
 
     # Handle vararg functions
+    prevararg_rewritten_atypes = atypes
```

Comment:
If this is what's used for the cache, is the below atypes actually used anywhere?

Reply:
Oh neat, you're right - the modified `atypes` was used elsewhere in the old inlining code, but not here. I guess I can just remove that code then.